scripts/validate_docstrings.py

```diff
@@ -455,6 +455,12 @@ def validate_one(func_name):
             if not rel_desc:
                 errs.append('Missing description for '
                             'See Also "{}" reference'.format(rel_name))
+
+    for line in doc.raw_doc.splitlines():
+        if re.match("^ *\t", line):
```

Comment:
'\t' in line is good enough, yes? we don't all *any* tabs

Reply:
If you see the comments above @TomAugspurger requested "Starts with tab" as there might be tabs to format a table or the like I think

Reply:
ok, not sure this is actually strict enough. as I said we don't allow *any* tabs

Reply:
I'm happy to change it back to allow no tabs at all. Let me know what you think @TomAugspurger

Reply:
@jreback There are cases where a `\t` is included in the text as literal character (for the explanation).
So I think only checking the start of a line is good.

Reply:
Unrelated to where `\t` is included, we could use `re.search(r'^ *\t', self.raw_doc, flags=re.MULTILINE)` and avoid the loop

Reply:
ups, sorry, I see now that you want to report the lines with the tabs...